python/xorbits/deploy/kubernetes/external_storage/juicefs/tests/test_juicefs.py

```diff
@@ -0,0 +1,166 @@
+import subprocess as sp
+
+import pytest
+
+from ....tests.test_kubernetes import (
+    _start_kube_cluster,
+    juicefs_available,
```

Comment:
This should not be imported from ``test_kubernetes``. This is just uesed in juicefs UT.